src/fileManagement.js

```diff
@@ -91,24 +102,15 @@ class Files {
     })
   }
 
-  /**
-   * Returns the url of a remote file - using the version 1 endpoint
-   * @param   {string}  path    path of the remote file at OC instance
-   * @returns {string}          Url of the remote file
-   */
-  getFileUrl (path) {
-    return this.helpers._buildFullWebDAVURL(path)
-  }
-
   /**
    * Returns the url of a remote file - using the version 2 endpoint
    * @param   {string}  path    path of the remote file at OC instance
    * @returns {string}          Url of the remote file
    */
-  getFileUrlV2 (path) {
+  getFileUrl (path) {
+    path = this._sanitizePath(path)
```

Comment:
```suggestion
    path = '/' + this._sanitizePath(path)
```
and drop the next line, since you implemented `_sanitizePath` to remove leading slashes.

Reply:
done